doc/xh.1

```diff
@@ -1,281 +1,397 @@
+
 Only necessary if the private key is not contained in the cert file.
-.HP
-\fB\-\-ssl\fR <VERSION>
-.IP
+.TP
+\fB\-\-ssl\fR=\fIVERSION\fR
 Force a particular TLS version.
-.IP
+
 "auto" gives the default behavior of negotiating a version with the server.
-.IP
+
 [possible values: auto, tls1, tls1.1, tls1.2, tls1.3]
-.HP
+.TP
 \fB\-\-native\-tls\fR
-.IP
 Use the system TLS library instead of rustls (if enabled at compile time)
-.HP
+.TP
 \fB\-\-https\fR
-.IP
 Make HTTPS requests if not specified in the URL
-.HP
-\fB\-\-http\-version\fR <VERSION>
-.IP
+.TP
+\fB\-\-http\-version\fR=\fIVERSION\fR
 HTTP version to use
-.IP
+
 [possible values: 1.0, 1.1, 2]
-.HP
+.TP
 \fB\-I\fR, \fB\-\-ignore\-stdin\fR
-.IP
 Do not attempt to read stdin
-.HP
+.TP
 \fB\-\-curl\fR
-.IP
-Print a translation to a `curl` command.
-.IP
+Print a translation to a curl command.
+
 For translating the other way, try https://curl2httpie.online/.
-.HP
+.TP
 \fB\-\-curl\-long\fR
-.IP
 Use the long versions of curl's flags
-.HP
-\fB\-\-help\fR
-.IP
-Print help information
-.HP
-\fB\-V\fR, \fB\-\-version\fR
-.IP
-Print version information
-.PP
-Each option can be reset with a \fB\-\-no\-OPTION\fR argument.
+.TP
+[\fIMETHOD\fR]\fI URL\fR
+The request URL, preceded by an optional HTTP method
+
+If the method is omitted, it will default to either GET or POST
+depending on whether the request contains body or not.
+
+    $ http example.com               # => GET
+    $ http example.com hello=world   # => POST
+
+Specifying the scheme portion of the request URL is optional.
+localhost can also be omitted from the URL as long it starts
+with colon plus an optional port number.
+
+    $ xh localhost:3000/users   # => http://localhost:3000/users
+    $ xh :3000/users            # => http://localhost:3000/users
+    $ xh :/users                # => http://localhost:80/users
+    $ xh example.com            # => http://example.com
+    $ xh ://example.com         # => http://example.com
+.TP
+[\fIREQUEST_ITEM\fR ...]
+Optional key\-value pairs to be included in the request
+
+The separator is used to determine the type i.e. header,
+request body, query string, etc
+
+    key==value
+        Add a parameter to the URL
+
+    key=value
+        Add a JSON field (\-\-json) or form field (\-\-form)
+
+    key:=value
+        Add a literal JSON value e.g. numbers:=[1,2,3]
+
+    key@file
+        include file in multipart request. Requires enabling
+        either \-\-form or \-\-multipart.
+
+        Additionally, both filename and mime type can be set e.g
+        picture@hello.jpg;type=image/jpeg;filename=goodbye.jpg
+
+    key=@file
+        Same as key=value but reads the value from a file
+
+    key:=@file
+        Same as key:=value but reads the value from a file
+
+    @filename
+        Use a file as the request body
+
+    header:value
+        Add a header
+
+    header:
+        Unset a header
+
+    header;
+        Add a header with an empty value
```

Comment:
For manpages, manually formatting strings like this instead of using a roff control is not a good idea as it would look bad on smaller screens.

I am yet to figure out how to use roff controls here without having two different help strings and risk them getting out of sync. However, if the two are defined in one place, maybe that would no longer be an issue? 

```rs
const REQUEST_ITEM_HELP: &'static str = "..";
const REQUEST_ITEM_MAN_HELP: &'static str = "..";
```